Python/Product/Profiling/Profiling.csproj

```diff
@@ -35,12 +35,22 @@
     <AssemblyName>Microsoft.PythonTools.Profiling</AssemblyName>
     <UseVSSDK>true</UseVSSDK>
     <CreateVsixContainer>true</CreateVsixContainer>
+    <DefineConstants Condition="$(IncludeExternalProfilerDriver) == 'true'">EXTERNAL_PROFILER_DRIVER;$(DefineConstants)</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Platform)' == 'x86' ">
     <PlatformTarget>x86</PlatformTarget>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="DiagnosticsHub.Packaging.Interop, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Is HintPath necessary? If so, then this won't work. If there's no NuGet package, we may have to check in the dlls into source control.

Reply:
Looks like `HintPath` is not needed, at least not in 15.8/Community, where I tested.  The removed property is in d618c1e0f.

Reply:
Can't find the relevant comment, but aa835d2d6 now removes the `public` qualifier from the types in `ExternalProfilerDriver` when not necessary.